.stylelintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "selector-type-case": "lower",
+    "selector-class-pattern": null,
+    "property-no-vendor-prefix": true
+  }
+}
```

Comment:
EOL

Reply:
In fact, I decided to do this in a new PR.😅